js/main.js

```diff
@@ -1,11 +1,34 @@
-function getWord(word) {
-  var xhr = new XMLHttpRequest();
-  xhr.open('GET', 'https://api.dictionaryapi.dev/api/v2/entries/en_US/' + word);
-  xhr.responseType = 'json';
-  xhr.addEventListener('load', function () {
-    console.log(xhr.status);
-    console.log(xhr.response);
+var $homelink = document.querySelector('.home-link');
+var $homePage = document.querySelector('.homepage-container');
+var $wordPage = document.querySelector('.wotd-container');
+var $navBar = document.querySelector('.nav-bar-container');
+var $wotdButton = document.querySelector('.wotd-button');
+var $newWordButton = document.querySelector('.new-word');
+
+function getRandomWord(event) {
+  var xhrRandomWord = new XMLHttpRequest();
+  xhrRandomWord.open('GET', 'https://random-word-api.herokuapp.com//word?number=1');
+  xhrRandomWord.responseType = 'json';
+  xhrRandomWord.addEventListener('load', function () {
+    var $wordOfTheDay = document.querySelector('.wotd');
+    $wordOfTheDay.textContent = xhrRandomWord.response[0];
   });
+  xhrRandomWord.send();
+}
 
-  xhr.send();
+function viewHomePage(event) {
+  $homePage.className = 'homepage-container';
+  $wordPage.className = 'wotd-container hidden';
+  $navBar.className = 'nav-bar-container hidden';
 }
+
+function viewWOTDPage(event) {
+  $homePage.className = 'homepage-container hidden';
+  $wordPage.className = 'wotd-container';
+  $navBar.className = 'nav-bar-container';
+  getRandomWord();
```

Comment:
You should make your API request before you show the page, and only show it when the page has loaded. When somebody clicks the link to move to the word of the day page, you should send out the request, and in the `load` event handler for that request, call this function to show the word view.